client/landing/gutenboarding/components/plans/plans-button/index.tsx

```diff
@@ -54,7 +55,7 @@ const PlansButton: React.FunctionComponent< Button.ButtonProps > = ( { ...button
 			<Button
 				onClick={ handleButtonClick }
 				label={ __( planLabel ) }
-				className="plans-button"
+				className={ classNames( 'plans-button', { 'user-selected': selectedPlan } ) }
```

Comment:
@razvanpapadopol said via Slack:

> `selectedPlan` is set only when a plan is explicitly selected. In order to have the button blue also when Premium Plan is displayed because of paid domain selection (like in Ola’s GIF) we need exactly what we have here: https://github.com/Automattic/wp-calypso/pull/42578/files#diff-a533017ba250fed92642a7cb13d6a7c6R48